pulsar recipe audit (#33)
startreedata/pinot-recipes: recipes/pulsar/README.md

> In this recipe we'll learn how to ingest data from Apache Pulsar.

This is the code for the following recipe: https://dev.startree.ai/docs/pinot/recipes/pulsar

***

## Makefile

```bash
make recipe
```

> To produce data to Pulsar, you can use the Python code below.

Reviewer: ```suggestion
To produce data into Pulsar, use the Python code below.
```
